src/main/java/org/broadinstitute/hellbender/tools/genome/SparkGenomeReadCounts.java

```diff
@@ -71,12 +69,41 @@
     )
     protected File outputFile;
 
-    private void collectReads(final JavaSparkContext ctx) {
+    /**
+     * Determine the intervals to consider for coverage collection.  Honors the keepAutosome parameter.
+     *
+     * <p>Developer's note:  This CLI will always set the attribute, intervals, to a non-null value.</p>
+     *
+     * @param rawIntervals Specified by the user.  If null, converts to SimpleIntervals specifying the entire
+     *                     reference genome.  If keepNonAutosomes is NOT specified, it will prune these intervals (or the
+     *                     ones specified by the user), to remove the contigs that are listed in the docs for
+     *                     {@link SparkGenomeReadCounts#keepNonAutosomes}
+     * @return Never {@code null}  Specified list of intervals.  These will be treated as if user had specified on the
+     *  CLI.
+     */
+    @Override
+    protected List<SimpleInterval> editIntervals(final List<SimpleInterval> rawIntervals) {
+        List<SimpleInterval> modifiedIntervals = rawIntervals;
+        if (rawIntervals == null) {
+            modifiedIntervals = IntervalUtils.getAllIntervalsForReference(getReferenceSequenceDictionary());
+        }
+
+        if (keepNonAutosomes) {
+            return modifiedIntervals;
+        }
+
+        // Enforce the elimination of certain contigs when proper option is set.
+        logger.info("Dropping non-autosomes, as requested...");
+        return modifiedIntervals.stream()
+                .filter(s -> !NONAUTOSOMALCONTIGS.contains(s.getContig()))
+                .filter(s -> !(s.getContig().startsWith("GL")) && !(s.getContig().startsWith("NC_")))
```

Comment:
It still seems a bit strange that it has an exclusion list rather than a human contigs inclusion list. Things like HLA contigs,  mitochondria, etc will pass this filter. Maybe doc should mention that it's assuming human hg19